.github/workflows/github-actions-ci.yml

```diff
@@ -1,22 +1,26 @@
 name: tests
+
 on: [push, pull_request]
+
 jobs:
   run-tests:
     runs-on: ubuntu-22.04
-    defaults:
-      run:
-        shell: bash -el {0}
+
     steps:
-      - name: checkout repository
-        uses: actions/checkout@v4.1.2
+      - uses: actions/checkout@v4
 
-      - name: create environment
-        uses: conda-incubator/setup-miniconda@v3
```

Comment:
Since we already use Miniconda as the environment for running our GDL scripts, should we also run the tests inside the same Miniconda environment? This would help ensure consistency between how the scripts are run and how they’re tested.